Sources/Deboogger.swift

```diff
@@ -176,6 +179,16 @@ public final class Deboogger {
         setup(recognizer, with: gesture)
     }
 
+    func setPerformanceMonitor(hidden: Bool) {
```

Comment:
Please, don't use `set` word in method. It would be better to use something like `updatePerformanceMonitor(hidden: Bool)`